spec/unit/crypto.spec.js

```diff
@@ -137,9 +138,15 @@ describe("Crypto", function() {
             )).client;
             await aliceClient.initCrypto();
             await bobClient.initCrypto();
+            // clobber the setTimeout function to run 10x faster.
+            realSetTimeout = global.setTimeout;
+            global.setTimeout = function(f, n) {
+                return realSetTimeout(f, n/10);
+            };
```

Comment:
generally the way you do this is by installing a fake clock with lolex which just runs timeouts immediately (unless you actually need the delay?)